test/intern/mixins/Tree.js

```diff
@@ -409,6 +409,22 @@ define([
 					}, null, 'Modification of child should not throw error');
 				});
 			});
+
+			test.test('collapse items removed from store', function() {
+				return expand(0).then(function() {
+					assert.isTrue(grid.row(0).element.classList.contains('dgrid-row-expanded'),
+						'Should have expanded class');
+					var item = grid.collection.getSync(0);
+					grid.removeRow(grid.row(0));
```

Comment:
This was verifying that the row was only expanded when the item was removed from the store as opposed to just from the grid. I removed it because this behavior should not have changed at all with this fix.